docker/services/api_compat/consul-template-api-compat.conf

```diff
@@ -4,8 +4,8 @@ template {
 }
 
 exec {
-    command = "uwsgi --die-on-term /etc/uwsgi/uwsgi-api-compat.ini"
-    splay = "60s"
+    command = "run-lb-command uwsgi --die-on-term /etc/uwsgi/uwsgi-api-compat.ini"
+    splay = "5s"
```

Comment:
60s was too long - it meant that a signal from consul-template could have taken up to 60s to get to our app